### PR TITLE
Handlebars

### PR DESCRIPTION
**Pull request by yevhensydorov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "express": "^4.15.4",
     "express-handlebars": "^3.0.0",
     "handlebars": "^4.0.11",
-    "knex": "0.13.0"
```
Comment (dahfool) [inline package.json:17]: is knex being used ?

File: index.js [hunk 2]
```diff
@@ -30,6 +31,23 @@ app.get("/", function(req, res, next) {
   res.render("home");
 });
 
+app.get("/reservations/:id?", function(req, res, next) {
```
Comment (dahfool) [inline index.js]: use arrow function.

File: index.js [hunk 3]
```diff
@@ -27,25 +27,21 @@ app.use("/api", apiRouter);
 // handle HTTP POST requests
 app.use(bodyparser.json());
 
+const filePath = __dirname + "/public/data/reservations.json";
```
Comment (dahfool) [inline index.js]: variable declaration at the top of file.

Comment (dahfool) [inline index.js]: use template literals rather than concatenation.

File: package.json (hunk 1 above)
Comment (yevhensydorov) [reply package.json:17]: I think we will use it in future. This package was here from the scratch

**Pull request merged**